src/pylibzfs2.h

```diff
@@ -115,6 +115,16 @@ typedef struct {
 	PyObject *name;
 } py_zfs_pool_t;
 
+typedef struct {
+	PyObject_HEAD
+	py_zfs_t *pylibzfsp;
```

Comment:
You can remove this if you put in pointer to originating `py_zfs_pool_t` object as mentioned below.